words2num/lang_EN_US.py

```diff
@@ -1,6 +1,7 @@
 from __future__ import division, unicode_literals, print_function
 import re
 from .core import NumberParseException, placevalue
+from decimal import Decimal, localcontext
```

Comment:
you forgot to commit this file

Reply:
`decimal` is a built-in Python library: https://docs.python.org/3/library/decimal.html